cirq/ops/pauli_string.py

```diff
@@ -344,14 +344,28 @@ def __str__(self) -> str:
 
         return prefix + '*'.join(factors)
 
+    def matrix(self,
+               qubits: Optional[Iterable[raw_types.Qid]] = None) -> np.ndarray:
```

Comment:
elsewhere we use the `QubitOrdering` class to have a more type-safe sentinel value for "default". That might be overkill here and/or not in the same spirit as this argument; just wanted to make a note of it

Reply:
In general, a `QubitOrder` describes a relation in a potentially unbounded set of qubits (see description of the second argument to `QubitOrder.explicit`), but we want the user to give us both: the set of qubits and their order, so this isn't a good fit.